lib/fileListFilters.ts

```diff
@@ -73,7 +73,7 @@ export interface IFileListFilter extends TypedEventTarget<IFileListFilterEvents>
 	 * @param nodes Nodes to filter
 	 * @return Subset of the `nodes` parameter to show
 	 */
-	filter(nodes: INode[]): INode[]
+	filter(nodes: Node[]): Node[]
```

Comment:
This is a breaking change and I do not see how it would help. 
(I requires the exact implementation with privat methods instead of the only required interface)

Reply:
Shall we use INode everywhere then? I find the multiple types confusing. 
Which are we supposed to use now ?